openxr/src/generated.rs

```diff
@@ -2960,6 +3029,38 @@ impl<'a> EyeCalibrationChangedML<'a> {
         (self.0).status
     }
 }
+#[derive(Copy, Clone)]
+pub struct LocalizationChangedML<'a>(&'a sys::EventDataLocalizationChangedML);
+impl<'a> LocalizationChangedML<'a> {
+    #[inline]
+    #[doc = r" # Safety"]
+    #[doc = r" `inner` must be valid event data according to the OpenXR spec. Refer to"]
+    #[doc = "[sys::EventDataLocalizationChangedML]"]
+    #[doc = r" for more information."]
+    pub unsafe fn new(inner: &'a sys::EventDataLocalizationChangedML) -> Self {
+        Self(inner)
+    }
+    #[inline]
+    pub fn session(self) -> sys::Session {
+        (self.0).session
+    }
+    #[inline]
+    pub fn state(self) -> LocalizationMapStateML {
+        (self.0).state
+    }
+    #[inline]
+    pub fn map(self) -> LocalizationMapML {
```

Comment:
This is a bit questionable because it contains a name field with a null terminated string. Should we put some effort into transforming it to a regular rust string?

Reply:
I think that'd be ideal. That might be why we don't generate the reexport by default.

Reply:
I made an attempt to fix it now. Please scrutinize appropriately!